.github/workflows/integration.yml

```diff
@@ -81,6 +81,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          version: 1.8.0
+          version: 2.4.18
```

Comment:
@bryantbiggs This too needs to be changed to 2.5.5 since it's changed to that in other places.